examples/subscribe_invoices.rs

```diff
@@ -0,0 +1,32 @@
+// This program accepts three arguments: address, cert file, macaroon file
+// The address must start with `https://`!
+
```

Comment:
It'd be nice to add some description for quick understanding. E.g.:

> This program connects to LND and prints out all incoming invoices as they settle.

Reply:
I followed you getinfo example which doesn't say  `This program connects to LND and prints out getinfo lnd node output`, you should update that one 😃

Reply:
I know, I will. :) It just occurred to me now that the description is missing.